test/helpers/secrets.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+/**
+ * A helper function to get secrets needed by tests
+ */
+function getTestSecret(secretName) {
+  const envVar = process.env[secretName]
```

Comment:
Wouldn't worry about it now but maybe a `.trim()` would be nice here eventually to prevent a certain brand of copy-paste error.

Reply:
Good call, I'll add that in